README.md

```diff
@@ -51,7 +51,7 @@ import qtawesome as qta
 
 ```python
 # Get icons by name.
-fa_icon = qta.icon('fa.flag')
+fa_icon = qta.icon('fa5s.flag')
```

Comment:
Why did you change everything to use `fa5s`? I'd prefer for you to use `fa5`.

Reply:
It was for visual consistency with your existing screenshot:
https://raw.githubusercontent.com/spyder-ide/qtawesome/master/qtawesome-screenshot.gif

In your README you use `fa.flag` which for FA 4.x is a solid looking flag:
https://fontawesome.com/v4.7.0/icon/flag
but then in FA 5.x, `fa5.flag` is the "regular" flag, which is rendered as an outline style:
https://fontawesome.com/icons/flag?style=regular
and the solid one got put in the "solid" style, which is `fa5s.flag`:
https://fontawesome.com/icons/flag?style=solid

This way, the icons in your example don't change visually and are only updated to the "modern" namespace of fa5s.

Do you still want me to use `fa5.flag` in all the examples?